shared/appengine.go

```diff
@@ -75,17 +76,16 @@ func (c *clientsImpl) Close() {
 	if c.cloudtasks != nil {
 		err := c.cloudtasks.Close()
 		if err != nil {
-			logrus.Warningf("Error closing cloudtasks: %s", err.Error())
+			log.Printf("Error closing cloudtasks: %s", err.Error())
```

Comment:
Is there a reason to use log vs logrus here?

Reply:
This is on the way out (exiting the program) and we can only log to stderr, so the logging level and formatting provided by logrus are less useful here (it'd end up as plaintext in stderr either way).